octavia-cli/octavia_cli/entrypoint.py

```diff
@@ -27,14 +33,14 @@ def octavia(ctx: click.Context, airbyte_url: str) -> None:
     ctx.obj["WORKSPACE_ID"] = workspace_id
 
 
-@octavia.command(help="Scaffolds a local project directories.")
-def init() -> None:
-    raise click.ClickException("The init command is not yet implemented.")
+def add_commands_to_octavia():
+    for command in AVAILABLE_COMMANDS:
+        octavia.add_command(command)
```

Comment:
Is there a reason why you defined this method to add commands to `octavia`, and then later call it, rather than just adding the commands directly in the `def octavia()` method?

Reply:
There could very well be some technical reason why you can't do the latter that I am unaware of!

Reply:
This a bit specific to the `Click` package we are using.
Here `octavia` is a command **group**, and `list` is a command group too.
It's the way I found to add a command group to another.

Usually to add a command (and not command group) to a group the decorator `@<command-group-name>.command()` works well, e.g to declare a `octavia create` command: 
```python 
@octavia.command(help="Generate a YAML configuration file to manage a resource.")
def create() -> None:
    ....
````

I could try if this work in `octavia-cli/octavia_cli/list/commands.py`:
```
@octavia.group
def _list():
  pass
```

But I like the fact of referencing everything that's added to the main `octavia` command to `octavia-cli/octavia_cli/entrypoint.py`, which will make it more readable to understand the flow.

Reply:
I see, I think that makes sense! I have no familiarity with the `Click` package, so I will defer to your judgement on the best approach to take here